### ISSUE TITLE
Simple by Unity link missing link content

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#12]
This is not technically true... screen readers will pick up the aria-label tag on the SVG. For some reason HTML_CodeSniffer still picks this up as a violation.

"Anchor element found with a valid href attribute, but no link content has been supplied."

**Comment by andrewmarino** [comment]
This is fixed a while ago!
